lightly/models/utils.py

```diff
@@ -463,28 +463,33 @@ def random_token_mask(
 
     return idx_keep, idx_mask
 
-def schedule_momentum(
-    iteration: int, 
-    max_iter: int, 
-    m: float = 0.99
+def cosine_decay_schedule(
+    step: int, 
+    max_steps: int, 
+    start_value: float = 0.996,
+    end_value: float = 1
```

Comment:
I wouldn't add default values here as the function is very general. It is, for example, also often used for learning rate schedules.